server/controllers/chartsController/helpers.test.js

```diff
@@ -153,6 +179,33 @@ describe('chartsController - helpers', () => {
         NY: { count: 0, targetTotal: 100 },
         Foo: { count: 0, targetTotal: 8 },
         Bar: { count: 0, targetTotal: undefined },
+        Totals: { count: 0, targetTotal: 128 },
+      })
+    })
+
+    it('Totals site will calculate correctly if a new site is introduced and a site is missing targets', () => {
```

Comment:
what does it mean to "calculate correctly"?

```suggestion
    it('generates empty value objects for new sites with missing targets', () => {
```

Reply:
Thank you for the feedback, I'll be more direct with the tests descriptions. 
In this case we don't calculate the N/A gray bar if a site from a chart is missing targets.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/230/commits/9577a75e9a4cfc69ba83f088b59cef0455ac950a